apps/src/p5lab/spritelab/commands/behaviorCommands.js

```diff
@@ -28,6 +29,27 @@ export const commands = {
     };
   },
 
+  followingTargetsFunc(p5Inst) {
+    return spriteArg => {
+      const sprite = coreLibrary.getSpriteArray(spriteArg)[0];
+      const spritePosition = sprite.position;
+      const targets = sprite.targetSet
+        .map(x => coreLibrary.getSpriteArray({costume: x}))
+        .flat();
+      // Find closest target
+      let closestTarget;
+      let closestDistance = Infinity;
+      targets.forEach(target => {
+        const distance = spritePosition.dist(target.position);
+        if (distance < closestDistance) {
+          closestDistance = distance;
+          closestTarget = target;
+        }
+      });
+      actionCommands.moveToward(spriteArg, 5, closestTarget.position);
```

Comment:
we should add a unit test to make sure `closestTarget` is always what we expect it to be, mostly so that it doesn't regress since the current calculation is pretty simple/solid